### PR TITLE
[browser] logging cleanup

### PR DESCRIPTION
**Pull request by pavelsavara**
- added `mono_log_info`, `mono_log_warn`, `mono_log_error` with forward to console
- added `mono_log_debug` which is silent unless `runtimeHelpers.diagnosticTracing` is set
- all of them prefix with `MONO_WASM: ` except `mono_log_info_no_prefix` which doesn't
- added new lint rule for `no-console`

### CODE REVIEW

File: src/mono/wasm/runtime/loader/logging.ts [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+        console.debug("MONO_WASM: " + msg, ...data);
+    }
+}
+
+export function mono_log_info(msg: string, ...data: any) {
+    console.info("MONO_WASM: " + msg, ...data);
+}
+
+export function mono_log_info_no_prefix(msg: string, ...data: any) {
+    console.info("MONO_WASM: " + msg, ...data);
```
Comment (ilonatommy) [inline src/mono/wasm/runtime/loader/logging.ts:18]: `all of them prefix with MONO_WASM:  except mono_log_info_no_prefix which doesn't`
here it's with the prefix

File: src/mono/wasm/runtime/debug.ts [hunk 2]
```diff
@@ -124,6 +126,7 @@ export function mono_wasm_raise_debug_event(event: WasmEvent, args = {}): void {
     if (typeof args !== "object")
         throw new Error(`args must be an object, but got ${JSON.stringify(args)}`);
 
+    // eslint-disable-next-line no-console
     console.debug("mono_wasm_debug_event_raised:aef14bca-5519-4dfe-b35a-f867abc123ae", JSON.stringify(event), JSON.stringify(args));
 }
```
Comment (radekdoulik) [inline src/mono/wasm/runtime/debug.ts:131]: Are the console calls here left on purpose?

Comment (pavelsavara) [reply src/mono/wasm/runtime/debug.ts:131]: they are somehow magical. @thaystg which of them could be removed or unified ?

**Review by radekdoulik — APPROVED** [review]
Besides the question about debug.ts it LGTM.

**Review by ilonatommy — APPROVED** [review]

File: src/mono/wasm/runtime/rollup.config.js [hunk 3]
```diff
@@ -375,5 +375,6 @@ function onwarn(warning) {
         return;
     }
 
+    // eslint-disable-next-line no-console
     console.warn(`(!) ${warning.toString()}`);
```
[the remark this answers is not in the document]
Comment (pavelsavara) [reply src/mono/wasm/runtime/rollup.config.js:379]: `crypto-worker.ts` is very old hash of main, it's gone now.
the other is `rollup.config.js`, that's tooling not production code.

File: src/mono/wasm/runtime/debug.ts (hunk 2 above)
Comment (pavelsavara) [reply src/mono/wasm/runtime/debug.ts:131]: I will merge this and we could improve debug usage of console in the future PRs.

**Pull request merged**